### PR TITLE
Project adviser bootstrap command

### PR DESCRIPTION
**Pull request by g4s8**
#833 - set project adviser in bootstrap command

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#839](https://github.com/zerocracy/farm/issues/839) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#839](https://github.com/zerocracy/farm/issues/839) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/781643318). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/main/resources/com/zerocracy/stk/pm/bootstrap.groovy [hunk 1]
```diff
@@ -62,16 +62,9 @@ def exec(Project project, XML xml) {
...
-   *  then bootstrap.groovy script should update catalog.xml with this parameter.
-   */
-  new Catalog(farm).bootstrap().link(
-    project.pid(),
-    'slack',
-    project.pid()
-  )
+  Catalog catalog = new Catalog(farm).bootstrap()
+  catalog.link(project.pid(), 'slack', project.pid())
+  catalog.adviser(project.pid(), claim.param('adviser'))
```
Comment (carlosmiranda) [inline src/main/resources/com/zerocracy/stk/pm/bootstrap.groovy]: @g4s8 let's add a new claim, "Adviser was updated".

**Comment by g4s8** [comment]
@carlosmiranda fixed in 93d8a71 please see new changes

**Comment by g4s8** [comment]
@yegor256 ping

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 11days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 11days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

File: src/main/resources/com/zerocracy/radars/q-project.xml [hunk 2]
```diff
@@ -160,6 +160,11 @@ SOFTWARE.
     <regex>bootstrap</regex>
     <label>bootstrap</label>
     <help>Bootstrap it right after adding Zerocrat to a Slack channel</help>
+    <opts>
```
Comment (yegor256) [inline src/main/resources/com/zerocracy/radars/q-project.xml]: @g4s8 I don't think we need this. The author of the claim is the adviser.

**Comment by yegor256** [comment]
@g4s8 see above

**Comment by g4s8** [comment]
@yegor256 fixed in 2540f5c

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 `SignatureTest` failed: https://github.com/zerocracy/farm/issues/853, please restart merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by carlosmiranda** [comment]
@yegor256 can we try this again?

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#839](https://github.com/zerocracy/farm/issues/839) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thanks!

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**